packages/reactstrap-validation-select/AvResourceSelect.js

```diff
@@ -15,7 +15,7 @@ class AvResourceSelect extends Component {
     }).isRequired,
     getResult: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
     delay: PropTypes.number,
-    label: PropTypes.string,
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
`PropTypes.node` includes `PropTypes.string`, so just setting it to `PropTypes.node` would be good enough.